.github/workflows/main.yml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: false
```

Comment:
Can this workflow be enabled again? What's the reason behind disabling it, is there no way to support it for the KMP project?